src/Apps/Collect/CollectionApp.tsx

```diff
@@ -125,6 +125,7 @@ export const CollectionAppFragmentContainer = createFragmentContainer(
         artworks(
           aggregations: $aggregations
           include_medium_filter_in_aggregation: true
+          sort: "-merchandisability"
```

Comment:
:ceiling_cat: Curious if this is intended or if we should be using `-decayed_merch` here (and in the other file). That's our standard sort on pages like `/collect`, and takes into account [other boosted parameters](https://github.com/artsy/gravity/blob/7b72b2766371890cc3702dbf5ed21feec0fa08ba/app/models/search/queries/filtered_query.rb#L25) aside from just merchandisability. (See [this slack convo](https://artsy.slack.com/archives/C9SATFLUU/p1562881100203900) for some extra context)